Header header
uint16 id
sensor_msgs/NavSatFix gps
mavros/State state
string type
bool ready
